alarm_control_panel-card/alarm_control_panel-card.js

```diff
@@ -323,17 +326,27 @@ class AlarmControlPanelCard extends HTMLElement {
     return style;
   }
 
-  _stateToIcon(state) {
-    return this._icons[state] || 'mdi:shield-outline'
-  }
+  _label(label, default_label=undefined) {
+    // Just show "raw" label; useful when want to see underlying const
+    // so you can define your own label.
+    if (this._config.show_labels === false) return label;
+
+    if (this._config.labels[label]) return this._config.labels[label];
```

Comment:
Also, as labels is optional, this will fail with JS error. Maybe:
```
    if (this._config.labels && this._config.labels[label]) return this._config.labels[label];
```

Reply:
Adding a translation for 'Clear' will cause `if (element.getAttribute('value') === 'Clear') {` to fail when pressing clear. 

Maybe update that to use:
```
      if (element.getAttribute('value') === this._label("ui.card.alarm_control_panel.clear_code")) {
```

Reply:
Great catches!